crates/tauri/src/app/plugin.rs

```diff
@@ -50,6 +52,11 @@ pub fn default_window_icon<R: Runtime>(
   })
 }
 
+#[command(root = "crate")]
+pub async fn set_app_theme<R: Runtime>(app: AppHandle<R>, theme: Option<Theme>) {
```

Comment:
This will cause a deadlock on plugins lock on these two places without `async`, I'm not sure how should we handle this, seems like it's because this call triggers a `ThemeChanged` event while having a lock on the plugins when invoked

https://github.com/tauri-apps/tauri/blob/f6a9717aadb8455f193fb35f20735f586d7dc2fb/crates/tauri/src/manager/mod.rs#L462

https://github.com/tauri-apps/tauri/blob/f6a9717aadb8455f193fb35f20735f586d7dc2fb/crates/tauri/src/app.rs#L2157